x-pack/plugins/security_solution/public/exceptions/manage_exceptions/use_create_shared_list.tsx

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ExceptionListSchema } from '@kbn/securitysolution-io-ts-list-types';
+
+import type { HttpStart } from '@kbn/core/public';
+import { useAsync, withOptionalSignal } from '@kbn/securitysolution-hook-utils';
+
+export const createSharedExceptionList = async ({
+  name,
+  description,
+  http,
+  signal,
+}: {
+  http: HttpStart;
+  signal: AbortSignal;
+  name: string;
+  description: string;
+}): Promise<ExceptionListSchema> => {
+  const res: ExceptionListSchema = await http.post<ExceptionListSchema>('/api/exceptions/shared', {
```

Comment:
Can we define a constant for `/api/exceptions/shared` I see it is used in the `route` as well

Reply:
Fixed here: f5f3ec39fea6438b0485f85e75afa38a8da209dc